tools/tokenserver/database.py

```diff
@@ -0,0 +1,641 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was adapted from [this file](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/assignment/sqlnode/sql.py) on the legacy Tokenserver. A good way to review this would probably be to look at the diff between the original version and my new version to see what changed